ps_calendar/templates/event_detail.html

```diff
@@ -80,20 +87,59 @@ <h5><b>TEXT</b></h5>
         </p>
     </div>    
 </div>
+
+<hr class="map" style="margin: 0;"/>
+<div id="map" style="margin-bottom: -10px;"></div>
```

Comment:
No inline css. Add extra class or id and add to css file.

Like the following:
```
hr.map {
    display: block;
    height: 1px;
    border: 0;
    border-top: 1px solid var(--color-primary);
    padding: 0;
    width: 100%;
    margin: 0px auto 10px;
  }

hr.map-event-detail {
    margin: 0;
}
```